### ISSUE TITLE
About training speed

### ISSUE THREAD

**Issue by MyeongJin-Kim** [RoyalVane/CLAN#7]
I found when train Deeplab from Imagenet pretrained weight, loss drop fast.
But when train FCN8s, loss drop extremely slow.
Did you get the same pattern?

**Comment by RoyalVane** [comment]
I also observed that pattern. I think it is reasonable to some degree since the residual network such as ResNet101 is easier to train than Vgg in my opinion. As a result I would spend more epochs to train FCN8s.
